profiles/nextstrain-public/rename.smk

```diff
@@ -0,0 +1,47 @@
+
+rule all_public:
+    input:
+        [
+            "auspice_renamed/" + build.get("auspice_name", f"{build_name}_{{segment}}").format(segment=segment) + suffix + ".json"
+            for build_name, build in config["builds"].items()
+            for segment in config["segments"]
+            for suffix in ["", "_root-sequence", "_tip-frequencies"]
+        ],
+
+def _get_file_by_auspice_name(wildcards):
+    for build_name, build_params in config["builds"].items():
+        for segment in config["segments"]:
+            if build_params.get("auspice_name", f"{build_name}_{{segment}}").format(segment=segment) == wildcards.auspice_name:
+                return f"auspice/{build_name}_{segment}.json"
+
+    return ""
+
+rule rename_auspice_main:
+    input:
+        _get_file_by_auspice_name,
+    output:
+        "auspice_renamed/{auspice_name}.json",
+    shell:
+        """
+        cp {input} {output}
```

Comment:
ICYMI, there was a recent [discussion in Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1673567129704069) about avoiding using `cp` in workflows. I don't think disk space will be too much of an issue with these JSONs, but switching to `ln` can improve runtime.

Reply:
Thanks! I had thought about using symlink here, but I forgot about hardlinking. Fixed in 817f197.